scripts/setup-ubuntu.sh

```diff
@@ -106,9 +107,6 @@ PACKAGES+=" zlib1g-dev"
 PACKAGES+=" libssl-dev:i386"
 PACKAGES+=" zlib1g-dev:i386"
 
-# So we don't build llvm for build.
-PACKAGES+=" llvm-8-tools"
-
 # For swift.
 PACKAGES+=" clang-9"
```

Comment:
Now that you added clang-10 above, you can remove this line and pull in my updated Swift commit that uses the newer clang, buttaface/termux-packages@222f67f.

Reply:
okay